### ISSUE TITLE
Dismiss callback/delegate

### ISSUE THREAD

**Issue by cikpis** [usabilla/usabilla-u4a-ios-swift-sdk#27]
Hi guys, is it possible to let devs handle the dismiss part? 

We are stuck in the app for that and I'm afraid we could not use this framework for it. It should be an easy fix and would be really nice to have.

Thanks in advance, cheers!

**Comment by gPinato** [comment]
Hey @cikpis ,

This feature was planned for the current sprint and it is scheduled to be released beginning of next week.
You will be able to disable the self dismiss of the form and dismiss it yourself once the user is done.

It is not possible to release it sooner than scheduled.

I will let you know as soon as the new version is available.

**Comment by cikpis** [comment]
@gPinato thanks for the effort guys.

Looking forward to your update, cheers!

**Comment by cikpis** [comment]
Hi @gPinato, how's it going? Will today be the day for the new release? :)

**Comment by gPinato** [comment]
@cikpis v3.3.0 has just been published. It contains the delegate call an should also fix the jump to top bug you're experiencing.

**Comment by cikpis** [comment]
@gPinato Cool! Thanks guys!

**Comment by cikpis** [comment]
Hi @gPinato, I opened another issue here https://github.com/usabilla/usabilla-u4a-ios-swift-sdk/issues/30
Could you take a look? Basically I cannot use the new version for an error.
